### ISSUE TITLE
Single quote in PR title breaks the get-pr-info variables

### ISSUE THREAD

**Issue by jakub-skierbiszewski-signavio** [NarrativeScience/circleci-orb-ghpr#8]
### Scenario
Given PR title:
> ASDF-111 don't use single quotes in PR title

When using step `ghpr/get-pr-info` and accessing `$GITHUB_PR_BASE_BRANCH` variable
Then variable is an empty line and error message is printed:
```
/tmp/.bash_env-5ff85b683890bc490b71ca94-0-build: line 4: unexpected EOF while looking for matching `''
/tmp/.bash_env-5ff85b683890bc490b71ca94-0-build: line 5: syntax error: unexpected end of file
```

### Details

I've noticed that exporting variables in `get-pr-info` command is not escaping the contents:
```
PR_TITLE=$(echo $PR_RESPONSE | jq -e '.title' | tr -d '"')
echo "PR_TITLE: $PR_TITLE"
echo "export GITHUB_PR_TITLE='$PR_TITLE'" >> $BASH_ENV
```
In my case this resulted in a corrupted env file:
```
echo $BASH_ENV
/tmp/.bash_env-5ff86be9b2b77a7bed582e27-0-build
cat $BASH_ENV
export GITHUB_PR_NUMBER=1568
export GITHUB_PR_TITLE='ASDF-111 don't use single quotes in PR title'
export GITHUB_PR_BASE_BRANCH='develop'
export GITHUB_PR_AUTHOR_USERNAME='jakub'
```
I think single quotes should be escaped to avoid such issues.

**Comment by CelestineKao** [comment]
Oh whoops, sorry I didn't see this earlier! I'll put in a fix shortly. 

Thanks for all the details 😄

**Comment by CelestineKao** [comment]
Hmm, so I'm not sure it's possible to do this actually. Or at least not without changing the outer enclosing `'` to `"`, which I'd prefer not to do to prevent variable expansion. 

From the [GNU docs](https://www.gnu.org/software/bash/manual/html_node/Single-Quotes.html):
> Enclosing characters in single quotes (‘'’) preserves the literal value of each character within the quotes. A single quote may not occur between single quotes, even when preceded by a backslash.

Open to suggestions though!

**Comment by jakub-skierbiszewski-signavio** [comment]
What about removing single quotes with `sed`?

**Comment by jakub-skierbiszewski-signavio** [comment]
Please have a look: https://github.com/NarrativeScience/circleci-orb-ghpr/pull/10

**Comment by jerbob** [comment]
This is still a problem when multiple single quotes are passed to the PR title or commit messages.
I have a PR up that should fix the problem: #15
